lib/constructs/dictionary.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-const conversions = require("webidl-conversions");
+const Types = require("../types.js");
```

Comment:
Since this is not a class, it should not be capitalized. Also, in this project it seems like we're not including `.js` in the requires.